data/helpers.d/logging

```diff
@@ -280,6 +282,7 @@ ynh_script_progression () {
 	fi
 
 	ynh_print_info "[$progression_bar] > ${message}${print_exec_time}"
+	set -x
```

Comment:
Globally agree with it, but don't you think this `set -x` should be just before the `ynh_print_info`, so we would have the echo into the log. Useful to know where the script is.
Didn't try though...

Reply:
Uh but the echo will be in the log as set +x/-x only affects the debug output of Bash

In fact, there will automatically be a set -x *inside* ynh_print_info because it relies on getopts, which itself also uses set -x

So from what I remember, the only usefulness of putting this set -x here (rather than before) is to hide a few additional `local foo` at the beginning of `ynh_print_info`

Reply:
Huh, is there any use of that last `set -x` as you said indeed, getops will do it.